### PR TITLE
Peer review and hosting of submitted projects. 

### PR DESCRIPTION
**Pull request by gregerhardt**
I am proposing a set of additions in an effort to open up the process more broadly.  Anyone can submit a project, and we would arrange for a review to ensure that it meets basic standards for openness and usability.  Basically, we want projects that someone projects that someone can actually download and use, and we are certifying that someone has tested that this is actually the case.  The organization would host the project, but not invest additional resources, and the author would continue to control it.  This 1) expands the range of tools that could be included with a limited investment of additional resources, and 2) provides a pool of projects from which to choose those we want to incubate.  

Please take a look, and comment/edit/reject as appropriate.

### CODE REVIEW

**Comment by e-lo** [comment]
So essentially like a TMIP peer-review?

File: 1.5-attributes-of-science.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+The scientific method is the process by which science is conducted, and is based on the elements of: question, hypothesis, prediction, testing and analysis (https://en.wikipedia.org/wiki/Scientific_method).  
+
+However, going back to Galileo's *Two New Sciences*, it has been recognized that even with a strong scientific process, additional community-level components are required to ensure the validity and credibility of science.  These include: 
+
+* Data recording and sharing
+* Reproducibility 
+* External review
+
+Scientific journals have long been central to ensuring that these community needs are met, by recording the data from experiments, sharing the methods of those experiments so they can be reproduced elsewhere, and providing a mechanism for external reviews.  However, as our data sets become larger, and our models more complex, the limits of the scientific paper as the basic unit of trade are increasingly clear.  While we do not challenge the core role that journals should continue to play, we also recognize that a complementary mechanism is needed to further promote these community scientific goals.  
+
```
Comment (e-lo) [inline 1.5-attributes-of-science.md]: ##TODO: insert examples/links to ways other academic communities have overcome this challenge [ i.e. the ArXive ]

File: 1.5-attributes-of-science.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+The emergence of open-source software and collaborative development sites, such as Github, are an important step towards moving beyond these challenges, because they ensure that the source code is recorded and open to external review and external use.  However, making the source code available is not a sufficient step to ensuring that a model can be shared and used in a practical sense.  This is for two reasons.  First, there is much more that goes into making a model run that simply having the code.  There are typically input data, parameters, and other materials that could be archived with the code, but often are not.  Second, in order to be usable in a practical sense, the model and the software must be specifically engineered to be shared.  Unless there is some mechanism to ensure that what is shared is not just available, but also accessible to an educated user, we may end up with nothing more than large amounts of spaghetti code.
```
Comment (e-lo) [inline 1.5-attributes-of-science.md]: I would go farther as to say that it isn't just about "running the model", but _using_ it.  This is where I think we can have the most improvement as a community the quickest acknowledging that there is only so much "science" that can be applied to predicting human behavior 20-30 years into the future.  I would expand on your last sentence about "educated user" means

**Comment by e-lo** [comment]
I'm going to merge this pull request so that it is not outstanding/open - with apologies if @DavidOry disagrees with it :-)

**Comment by DavidOry** [comment]
Thrilled to have someone else merge the request.  Thanks @gregerhardt for the contribution and @e-lo for the thoughtful comments.

**Pull request merged**